app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/api/MvvApiService.kt

```diff
@@ -0,0 +1,72 @@
+package de.tum.`in`.tumcampusapp.component.ui.transportation.api
```

Comment:
Would this maybe not be better in the API package, where we have the other APIs?